pom.xml

```diff
@@ -82,6 +82,16 @@
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
             <version>2.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <version>1.0.0.GA</version>
```

Comment:
I have a feeling that this dependency is wrong, check this one https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-validation

Reply:
thx